Base/Python/slicer/tests/test_slicer_parameter_node_wrapper.py

```diff
@@ -525,6 +531,102 @@ class ParameterNodeType:
         param2 = ParameterNodeType(param.parameterNode)
         self.assertEqual(param2.p, [[4, 5, 0, 7]])
 
+    def test_tuple(self):
+        @parameterNodeWrapper
+        class ParameterNodeType:
+            a: tuple[Annotated[int, Minimum(0)], str]
+            b: Annotated[tuple[float, bool], Default((44.0, False))]
+
+        param = ParameterNodeType(newParameterNode())
+        self.assertTrue(param.isCached("a"))
+        self.assertTrue(param.isCached("b"))
+
+        self.assertEqual(param.a, (0, ""))
+        self.assertEqual(param.b, (44.0, False))
+
+        param.a = (1, "a")
+        param.b = (4.4, False)
+
+        self.assertEqual(param.a, (1, "a"))
+        self.assertEqual(param.b, (4.4, False))
+
+        with self.assertRaises(ValueError):
+            param.a = (-2, "hi")
+
+    def test_tuple_of_custom(self):
+        @parameterNodeWrapper
+        class ParameterNodeType:
+            a: tuple[CustomClass, int]
+
+        param = ParameterNodeType(newParameterNode())
+        self.assertFalse(param.isCached("a"))
+
+        self.assertEqual(param.a, (CustomClass(0, 0, 0), 0))
+
+        param.a = (CustomClass(7, 6, 5), 9)
+        self.assertEqual(param.a, (CustomClass(7, 6, 5), 9))
+
+    def test_dict(self):
+        @parameterNodeWrapper
+        class ParameterNodeType:
+            a: dict[str, int]
```

Comment:
The same comments about mutable-valued dictionaries; maybe `dict[str, list[int]]` or a nested dict.

Reply:
"The same comments" referring to the comments on `TupleSerializer` - https://github.com/Slicer/Slicer/pull/6727#discussion_r1044876943

Reply:
Added test with dictionary of lists.